### DISCUSSION TITLE
[v 1.21.7] All clients on a server continuously receiving hints of coordinates

### DISCUSSION THREAD

**Discussion by cmbtnt** [Vdauphin/HeartsAndMinds#1217] [Question and answer] [answered]
Hello, I have run into an issue where slightly after initialization, every client on the server is spammed with what I assume are debug messages in the form of hints (attached image example).

![107410_20211029165523_1](https://user-images.githubusercontent.com/46329775/139513762-339f2fc9-2545-438b-af22-307ef94b0532.png)

The issue appears to be clientside, as each client starts receiving the messages about three minutes after they load in. It starts with one message, and then after about 1-2 minutes the client is absolutely flooded with hundreds of hints. Everyone is also experiencing significant FPS lag. I have tweaked both the mission.sqf and param.hpp files, as well as a small number of files to change some of the objects spawned (landmines, flag at hideouts, potentially other hideout objects). I did test those changes and they appear to be working fine. In our attempts to debug the mission, we've considered the following factors:

- Our server does not have a headless client
- I changed the playerslots to OPFOR (side_mission, btc_player_side variables accounted for)
- Simplex Support AIs are placed down in the mission, and their vehicles are not defined as btc_veh's

An interesting quirk we noticed was that when only the adminLogged player joined, there were no errors. This is my first time editing this mission for a dedicated server, so it is possible something is set up wrong server-side. I have looked through the server logs and there does not appear to be anything of note, but I can upload them for you if you like.

Thanks

**Comment by Vdauphin** [comment]
Hello,
This hint have a position coordinate format. H&M doesn't have any hint in this format. Also, if you didn't activate the debug mode in H&M parameter, you should see no debug message. I am pretty sure this is not from the mission.

Try to ask this to author of scripts or mods you are using
Cheers

**Reply by cmbtnt** [reply]
That's something I'll look at, but we have not substantially changed our modlist in some time and only have encountered this error when we started using Hearts & Minds. Are there any mods/scripts that have been noted to be problematic in the past? Is there a method you can recommend so I can determine what is creating this hint?

Thanks

**Reply by Vdauphin** [reply]
> Is there a method you can recommend so I can determine what is creating this hint?

Search for `hint` in your mission

**Reply by Vdauphin** [reply]
> Are there any mods/scripts that have been noted to be problematic in the past?

None

**Reply by cmbtnt** [reply]
It is likely that [Pook Soviet Air Force Pack](https://steamcommunity.com/sharedfiles/filedetails/?id=1595678775&searchtext=pook+soviet+afl) is the issue. I have not observed the issue when it is unloaded locally, but we have not tested it on the dedicated yet.

**Comment by cmbtnt** [comment, marked as answer]
Apologies for the long delay, I forgot to update this issue. [Pook Soviet Air Force](https://steamcommunity.com/sharedfiles/filedetails/?id=1595678775) was in fact the issue. An extremely strange error to say the least.
